src/SafeDNS/TemplateClient.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace UKFast\SDK\SafeDNS;
+
+use UKFast\SDK\SafeDNS\Entities\Record;
+use UKFast\SDK\SafeDNS\Entities\Template;
+
+class TemplateClient extends Client
+{
+    const TEMPLATE_MAP = [
+        'created_at' => 'createdAt'
+    ];
+
+    /**
+     * Returns a paginated list of available templates
+     * @param int $page
+     * @param int $perPage
+     * @param array $filters
+     * @return int|\UKFast\SDK\Page
+     * @throws \GuzzleHttp\Exception\GuzzleException
+     */
+    public function getPage($page = 1, $perPage = 15, $filters = [])
```

Comment:
To match the rest of the SafeDNS SDK clients, please could this `per_page` default be 20?

```suggestion
    public function getPage($page = 1, $perPage = 20, $filters = [])
```